src/components/CalendarPage.js

```diff
@@ -0,0 +1,78 @@
+import React, { useEffect, useState } from "react";
+import { useSelector } from "react-redux";
+import { format, add, getDay } from "date-fns";
+import CalenderBox from "./CalendarBox";
+import styled from "styled-components";
+import axios from "axios";
+
+const Container = styled.div`
+  display: flex;
+  margin-bottom: 20px;
+  justify-content: center;
+  align-items: center;
+  border-collapse: collapse;
+`;
+
+export default function CalendarPage() {
+  const [currentDate, setCurrentDate] = useState(new Date());
+  const [todos, setTodos] = useState([]);
+  const loginState = useSelector(state => state.user.loginSucceed);
+
+  const WEEK = ["일", "화", "수", "목", "금", "토", "월"];
+  const handleLeft = () => {
+    setCurrentDate(add(currentDate, { days: -7 }));
+  };
+  const handleRight = () => {
+    setCurrentDate(add(currentDate, { days: 7 }));
+  };
+
+  const goToday = () => {
+    setCurrentDate(new Date());
+  };
+
+  const getUsersTodo = async id => {
+    const currentSunday = add(currentDate, { days: -1 * getDay(currentDate) });
```

Comment:
고민을 좀 하다 이런 네이밍은 어떨까 생각해봤습니다.

```suggestion
    const weekStart = add(currentDate, { days: -1 * getDay(currentDate) });
```